packages/rich-text/src/create.js

```diff
@@ -138,22 +137,15 @@ export class RichTextData {
 		return richTextData;
 	}
 	constructor( init = createEmptyValue() ) {
-		// Setting text, formats, and replacements as enumerable properties
-		// unfortunately visualises these in the e2e tests. As long as the class
-		// instance doesn't have any enumerable properties, it will be
-		// visualised as a string.
-		Object.defineProperty( this, RichTextInternalData, { value: init } );
+		this.#value = init;
 	}
 	toPlainText() {
-		return getTextContent( this[ RichTextInternalData ] );
+		return getTextContent( this.#value );
 	}
 	// We could expose `toHTMLElement` at some point as well, but we'd only use
 	// it internally.
 	toHTMLString() {
-		return (
-			this.originalHTML ||
-			toHTMLString( { value: this[ RichTextInternalData ] } )
-		);
+		return this.originalHTML || toHTMLString( { value: this.#value } );
```

Comment:
The way how `originalHTML` is defined in `fromHTMLElement` is not entirely correct. It effectively creates a _live binding_ between the `htmlElement` and the `RichTextData` value. If the element's contents change, then `toHTMLString()` will read the latest `.innerHTML` value in the getter.

Other parts of the `RichTextData` value are however not live: the `toPlainText` method, the `.text` getter, etc. The object's content can be inconsistent and out of sync.

Reply:
RichTextData should not be mutable

Reply:
Then we need to read `htmlElement.innerHTML` only once in `RichTextData` constructor, not on every call of `.toHTMLString`.

Reply:
We do it only once in the static method? `originalHTML` is a value not a getter?

Reply:
(We only want to serialise when it's needed)

Reply:
> `originalHTML` is a value not a getter?

Oh, somehow I overlooked this 🙂 It's alright then.